### ISSUE TITLE
Pacstrap error:

### ISSUE THREAD

**Issue by rohandebsarkar** [archlinux/archinstall#785]
![VirtualBox_arch1_04_12_2021_09_23_27](https://user-images.githubusercontent.com/50144004/144695987-ecdafe16-ea4e-4bce-a7c7-cf871dd4bf75.png)

**Comment by Torxed** [comment]
Interesting issue. I think archinstall did what we expect it to. It listened to the `pacman` and `pacstrap` exit codes and aborts on failure. In the future we'll implement re-trying of strapping in packages.

But the error itself is interesting, `--noconfirm` disables any interaction with `pacstrap`, but it seams as if it chose `(default)` instead of `all` on the group selection for `base-devel`. If you check `/var/log/archinstall/cmd_history.log` I'm pretty sure the command will be a standard `pacstrap --noconfirm /mnt/archinstall base base-devel linux linux-firmware`. If that's the case I would probably suggest we file a bug upstream on `pacstrap`.

**Comment by chung1905** [comment]
The command is:
```
/usr/bin/pacstrap /mnt/archinstall base base-devel linux-firmware linux (default) intel-ucode --noconfirm
```

**Comment by Torxed** [comment]
Good catch @chung1905! I'll get on to fix that one!

**Comment by Torxed** [comment]
Fixed in #790. It was an issue with the way `menu.py` returned the default option when one was given.
